dimagi/pages/static/style/pages/blog.scss

```diff
@@ -15,7 +15,6 @@
 a.post {
   color: $color-text !important; // scss-lint:disable ImportantRule
   font-weight: $font-weight-regular;
-
```

Comment:
please insert the line between definitions back. don't delete them. there shouldn't be any change in the `blog.scss` file